nsenter/event.go

```diff
@@ -616,6 +620,34 @@ func (e *NSenterEvent) processMountSyscallRequest() error {
 
 	payload := e.ReqMsg.Payload.([]domain.MountSyscallPayload)
 
+	if payload[0].FsType == "overlay" {
```

Comment:
Let's add a comment indicating why specifically for overlayfs mounts the handler needs to adopt the uid:gid of the container process that performed the mount syscall.

Reply:
done.